user-guide/Advanced_Functionality/Swarming/Messages/SwitchingObjects.md

```diff
@@ -0,0 +1,22 @@
+---
+uid: SwitchingObjects
+---
+
+# Switching objects
+
+With DataMiner Swarming, you can switch any [DataMiner object](xref:BasicConcepts#system-components) from one DataMiner Agent in a cluster to another.
+
+## Switching an element
```

Comment:
I've incorporated your original page as a subsection within the new and broader 'Switching Objects' page. This will later make it simpler to build upon the information we already have. Now that we have one subpage underneath the main 'Swarming' page, we can also add several more on different swarming-related topics.